### PR TITLE
Update config settings for Docker Compose

### PR DESCRIPTION
**Pull request by matea16**
### Description

Update configuration settings for Docker Compose

### Pull request type

Please check what kind of PR this is:

- [x] Fix or improvement of an existing page
- [ ] New documentation page, release related

Closes:
https://github.com/memgraph/documentation/issues/670
https://github.com/memgraph/documentation/issues/671

### Checklist:

- [ ] Check all content with Grammarly
- [ ] Perform a self-review of my code
- [ ] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [ ] The build passes locally
- [ ] My changes generate no new warnings or errors
- [ ] Add a corresponding label
- [ ] If release-related, add a product and version label
- [ ] If release-related, add release note on product PR

### CODE REVIEW

File: pages/configuration/configuration-settings.mdx [hunk 1]
```diff
@@ -103,7 +103,80 @@ Memgraph with **Docker**, **Docker Compose**, or if it was installed on the nati
...
+        volumes:
+          mg_lib:
+          mg_log:
+          mg_etc:
+        ```
+        ---
+
+        You can also change the configuration within the configuration file:
+
+        <Steps>
```
[the remark this answers is not in the document]
Comment (katarinasupe) [reply pages/configuration/configuration-settings.mdx:135]: Agreed.

File: pages/configuration/configuration-settings.mdx [hunk 2]
```diff
@@ -365,7 +438,42 @@ In order to apply the environment variables with Memgraph, you can follow the st
...
+          mg_lib:
+          mg_log:
+          mg_etc:
+        ```
+        ---
+
+        After you have added the environment variables, restart the instance by
+        running the following command:
+        ```
+        docker restart <CONTAINER_ID>
```
[the remark this answers is not in the document]
Comment (matea16) [reply pages/configuration/configuration-settings.mdx:468]: I tested it out and you're right, it should be docker down and up, I'll adjust this

File: pages/configuration/configuration-settings.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (antejavor) [reply pages/configuration/configuration-settings.mdx:135]: @katarinasupe should handle the Docker version.

File: pages/getting-started/install-memgraph/docker-compose.mdx [hunk 3]
```diff
@@ -65,15 +66,34 @@ services:
...
-### Notes
+{<h3>Configuration</h3>}
 
 - This configuration is primarily optimized for development and testing
   environments. For production deployments, it's recommended to customize the
   configuration according to specific security and performance requirements.
   
-- The `--log-level=TRACE` option in the Memgraph service enables detailed
+- In this Docker Compose file, the `command` directive is used to specify the
+  configuration flags  that should be set when the container starts. In this
```
Comment (katarinasupe) [inline pages/getting-started/install-memgraph/docker-compose.mdx:76]:
```suggestion
  configuration flags that should be set when the container starts. In this
```

**Review by katarinasupe — APPROVED** [review]

**Pull request merged**